types/index.d.ts

```diff
@@ -159,7 +159,8 @@ export interface Column<RowData extends object> {
   customSort?: (
     data1: RowData,
     data2: RowData,
-    type: "row" | "group"
+    type: "row" | "group",
+    isDesc: "desc" | undefined
```

Comment:
Thanks for your effort. But i think it was a mistake. Because we don't need to pass isDesc parameter to customSort function. Because we already check for sort direction before calling.